### DISCUSSION TITLE
LEO - MEO - GEO statistics

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1872] [Ideas / Feature Requests]
Hi guys,

Is it possible to have separate statistics (VUCC, DXCC, anything else?) for LEO, MEO, GEO sats?
For example, to put checkmark next to "Show LEO" and then we got all those nice number related only to LEO sats...
Or am I missing this feature in the most recent version of CL (IO-117 added, but not yet FO-118)?

Regards,
Dragan, 4O4A

**Comment by phl0** [comment]
Hej Dragan, I already had that idea in mind to distinguish LEO and GEO. Should be adaptable to include MEO as well. Not yet ready though. 
But latest Cloudlog has support for IO-117 and FO-118 already.

**Reply by 4O4A** [reply]
Thanks, Flo. 
My bad in original post -- FO-118 *is* there. Updated today afternoon.
Keep up good work!
73
